coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/config/SecondaryConfig.scala

```diff
@@ -8,10 +8,11 @@ import net.ceedubs.ficus.FicusConfig._
 // reflection-based horror that doesn't even provide full paths to
 // errors.
 
+case class StoreConfig(storeCapacityMB: Long, acceptingNewDatasets: Boolean)
+
 case class SecondaryGroupConfig(
      numReplicas: Int,
-     instances: Set[String],
-     instancesNotAcceptingNewDatasets: Option[Set[String]]
+     instances: Map[String, StoreConfig]
```

Comment:
@znep You may have feelings about this config change.

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/38a6faf8-807d-4ae3-8159-8d2659b88cec-69d0c8e1-e986-47fb-a727-1a8ae0328e06-v1.png?transparent=1&palette=1&width=246)

Reply:
What does that mean?